src/app/cpu-utilization-monitor/cpu-observer-list/cpu-observer-list.component.ts

```diff
@@ -1,24 +1,25 @@
-import { Component, OnInit } from '@angular/core';
-import { CpuObservationEndpoint } from 'cpu-monitoring-models';
-import { EndpoitsService } from '../endpoits.service';
+import { Component } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
+import { CpuObservationEndpoint } from 'cpu-monitoring-models';
 import { EditObservationEndpointComponent } from '../edit-observation-endpoint/edit-observation-endpoint.component';
+import { EndpoitsService } from '../endpoits.service';
 
 @Component({
   selector: 'app-cpu-observer-list',
   templateUrl: './cpu-observer-list.component.html',
   styleUrls: ['./cpu-observer-list.component.scss']
 })
 /**
- * component to add a cpu observation endpoint to a list
+ * component to add a cpu observation endpoint to list of observed endpoints
  */
 export class CpuObserverListComponent {
 
-  constructor(public endpointService: EndpoitsService, private matDialog: MatDialog) {}
+  constructor(public endpointService: EndpoitsService, private matDialog: MatDialog) { }
 
   /**
-   * create an CpuObservationEndpoint object and send it via post request to the endpoint
-   * @param endpoitData CpuObservationEndpoint object as an observation endpoint
+   * creates a  new CpuObservationEndpoint object and hands it to endpointsService where its saved and can be started for observation.
+   * 
+   * @param endpointData CpuObservationEndpoint object as an observation endpoint
```

Comment:
mein vorschlag wäre
```suggestion
   * @param endpointData  endpoint to be added to the observed enpoints
```
deine version ist ziemlich nicht sagen. dass ein observation endpoint ein observation endpoint ist, ist auch so klar.